### PR TITLE
show invited users who doesn't have yet accepted the invitation

### LINKED ISSUE #1505: PrintView: add invited collaborators
OJ: When countries adds collaborators and the collaborator never accessed (never accepted the invitation) the platform then the collaborators are not visible in the printed version of the report. Some countries has issues with this, think Cuba was one of the. (TODO: ADD ALSO INVITED COLLABORATORS TO THE PRINTED VERSIONS)

### PR DESCRIPTION
**Pull request by rfontanarosa**
close #1505

- The one with 1a is a user with status = active invited as a Collaborator for that Country and the Table/Section 1a
- The one with None is a user with status = pendingInvitation,  invited as a Collaborator for that Country and without any Table/Section permissions

The previous condition was:
- User status must be active
- User must not be a Reviewer
- User must have assigned a Section

In the new condition I have removed the third point

Schermata 2022-09-16 alle 11 29 23

### CODE REVIEW

File: src/client/pages/AssessmentPrint/FraPrint/ContactPersons/ContactPersons.tsx [hunk 1]
```diff
@@ -31,18 +31,14 @@ const ContactPersons = () => {
       }
     }, {})
 
-  const users = useUsers().filter(
-    (user) =>
-      user.status === UserStatus.active &&
-      !Users.isReviewer(user, countryIso) &&
-      user.roles?.[0]?.props?.sections !== 'none'
-  )
+  const users = useUsers().filter((user) => user.status === UserStatus.active && !Users.isReviewer(user, countryIso))
```
Comment (minotogna) [inline src/client/pages/AssessmentPrint/FraPrint/ContactPersons/ContactPersons.tsx:34]: this doesn't seem right.

**Review by minotogna — CHANGES_REQUESTED** [review]

File: src/client/pages/AssessmentPrint/FraPrint/ContactPersons/ContactPersons.tsx (hunk 1 above)
Comment (rfontanarosa) [reply src/client/pages/AssessmentPrint/FraPrint/ContactPersons/ContactPersons.tsx]: gotcha

File: src/server/api/user/getMany.ts [hunk 2]
```diff
@@ -6,18 +6,21 @@ import { AssessmentController } from '@server/controller/assessment'
...
     const { assessment, cycle } = await AssessmentController.getOneWithCycle({ assessmentName, cycleName })
 
-    const users = await UserController.getMany({
+    let users = await UserController.getMany({
       countryIso,
       assessment,
       cycle,
     })
 
+    if (print && print === 'true')
```
Comment (sorja) [inline src/server/api/user/getMany.ts:21]: Enough to check `print === 'true'`? undefined !== 'true'

**Review by sorja — APPROVED** [review]

File: src/server/api/user/getMany.ts [hunk 3]
```diff
@@ -6,18 +6,21 @@ import { AssessmentController } from '@server/controller/assessment'
...
+      users = users.filter((user) => !(process.env.FRA_REPORT_COLLABORATORS_EXCLUDED ?? []).includes(user.email))
```
Comment (minotogna) [inline src/server/api/user/getMany.ts:22]: usare ProcessEnv (aggiungere a ProcessEnv la proprietà fraReportCollaboratorsExcluded = JSON.parse(process.env.FRA_REPORT_COLLABORATORS_EXCLUDED??'[]')

**Review by minotogna — CHANGES_REQUESTED** [review]

**Review by minotogna — APPROVED** [review]

**Pull request merged**